presto-main/src/main/java/com/facebook/presto/operator/aggregation/noisyaggregation/sketch/SfmSketch.java

```diff
@@ -203,7 +203,7 @@ public Bitmap getBitmap()
 
     private static RandomizationStrategy getDefaultRandomizationStrategy()
     {
-        return new SecureRandomizationStrategy();
+        return new ThreadLocalRandomizationStrategy();
```

Comment:
This should not be the default. We should use SecureRandom in all cases except one (which does not require it): merging sketches.

Consequently, this change should only really affect the merge() and merge_sfm() functions. (The other aggregators do use merges, but only on noiseless sketches anyway, which are deterministic operations.)